vite.base.config.ts

```diff
@@ -0,0 +1,28 @@
+import plainText from 'vite-plugin-plain-text';
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  plugins: [
+    plainText('**/*.hbs', {
+      namedExport: false,
+    }),
+  ],
+  test: {
```

Comment:
The default timeout of vitest is 5s, but our previous jest configuration had a 15s timeout.
```suggestion
  test: {
    testTimeout: 15000,
```

Reply:
We want this timeout to be as small as possible, as it helps bring slower tests to light.

This should be increased only when strictly _necessary_ but avoided as much as _possible_.

The fact it is working with `5s` instead of `15s` is a huge win.

If it could be less than `5s`, it should.

Reply:
I'd be keen to leave it as is. Even in CI the longest running `fuel-gauge` tests only tend to be running around `2s`.